packages/assets/scss/00-base/_fonts-langs-vars.scss

```diff
@@ -31,8 +31,9 @@ $japanese:				U+A5, U+4E00-9FFF, U+30??, U+FF00-FF9F; // Kanji, Hiragana, Kataka
 $kannada:					U+0C80-0CFF;
 $kayah:						U+A900-A92F;
 $khmer:						U+1780-17FF, U+19E0-19FF; // Khmer and symbols
-$lao:							U+0E80-0EFF;
-$latin:						U+????;
+$lao:							U+????; // U+0E80-0EFF;
```

Comment:
@clairesunstudio Just wanted to check whether this is intentional or was instead intended as a placeholder that didn't get follow up